_data/beyond-sidebar.yml

```diff
@@ -559,9 +576,12 @@
             - title: Capture payment
               link: https://api-docs.beyondshop.cloud/ng-order-order-public-api.html#resources-payment-process-capture
               id: capture_payment
-            - title: Create pickup processes
+            - title: Create pickup process
               link: https://api-docs.beyondshop.cloud/ng-order-order-public-api.html#resources-pickup-processes-create
               id: create_pickup_processes
+            - title: Create multiple pickup processes
+              link: https://api-docs.beyondshop.cloud/ng-order-order-public-api.html#resources-order-batch-actions-create-packingslips-pickup
```

Comment:
I wonder if this can still be changed, but it looks a bit confusing to have the resource link indicate "create packingslips", but the title only mentions pickup processes...?

Reply:
Due to the fact that the individual action is "Create pickup process" I think we should keep it that way. If I'm not mistaken this won't be display publicly. It's just the name of the anchor the sidebar is using to connect to the resource. And the batch action is actually creating multiple packing slips but the pickup/shipping processes are created at the same time.